client/client_test.go

```diff
@@ -0,0 +1,190 @@
+package client
+
+import (
+	"context"
+	"errors"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/rs/zerolog"
+)
+
+func TestNew(t *testing.T) {
```

Comment:
Could this test be renamed?

Reply:
It is named after the function it references which is 'New' and is apparently following idiomatic naming conventions!